resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/SparkPodInitContainerBootstrap.scala

```diff
@@ -16,8 +16,12 @@
  */
 package org.apache.spark.deploy.k8s
 
-import io.fabric8.kubernetes.api.model.{ContainerBuilder, EmptyDirVolumeSource, PodBuilder, VolumeMount, VolumeMountBuilder}
+import scala.collection.JavaConverters._
 
+import io.fabric8.kubernetes.api.model._
```

Comment:
Are these imports supposed to be wildcarded?

Reply:
Not sure about the rule. The IDE automatically did the change. Reverted.